go-app-ussd_clinic.js

```diff
@@ -115,6 +146,14 @@ go.app = function() {
                 registrant_info.details.source = "clinic";
             }
 
+            if (registrant_info.details.clinic) {
+                registrant_info.details.clinic.redial_sms_sent = self.im.user.answers.redial_sms_sent;
+            } else {
+                registrant_info.details.clinic = {};
+                registrant_info.details.clinic.redial_sms_sent = self.im.user.answers.redial_sms_sent;
```

Comment:
You could just put this in the dict in the line above to make it a bit clearer, i.e.
`registrant_info.details.clinic = {redial_sms_sent: self.im.user.answers.redial_sms_send}`